scripts/box

```diff
@@ -304,19 +292,20 @@ function _adduser() {
     done
 
     USER=${user^^}
-    cat > /etc/sudoers.d/${user} << SUD
+
+    #shellcheck source=sources/functions/utils
+    . /etc/swizzin/sources/functions/utils
+    cat > /etc/sudoers.d/"${user}" << SUD
 Cmnd_Alias   ${USER}CMDS = /bin/systemctl stop flood@${user}, /bin/systemctl restart flood@${user}, /bin/systemctl start flood@${user}, /bin/systemctl start rtorrent@${user}, /bin/systemctl stop rtorrent@${user}, /bin/systemctl restart rtorrent@${user}, /bin/systemctl start deluged@${user}, /bin/systemctl stop deluged@${user}, /bin/systemctl restart deluged@${user}, /bin/systemctl start deluge-web@${user}, /bin/systemctl stop deluge-web@${user}, /bin/systemctl restart deluge-web@${user}, /bin/systemctl start irssi@${user}, /bin/systemctl stop irssi@${user}, /bin/systemctl restart irssi@${user}, /bin/systemctl start qbittorrent@${user}, /bin/systemctl stop qbittorrent@${user}, /bin/systemctl restart qbittorrent@${user}, /usr/local/bin/swizzin/box chpasswd ${user}
 
 ${user}     ALL = (ALL) NOPASSWD: ${USER}CMDS
 SUD
```

Comment:
We don't want this to run if the user is master as they get full sudo permissions

Reply:
Aren't these the sudo permissions to run their own tasks?